cfme/tests/networks/test_sdn_navigation.py

```diff
@@ -21,11 +21,8 @@ def test_sdn_provider_relationships_navigation(provider, tested_part, appliance)
     Metadata:
         test_flag: sdn
     """
-    view = navigate_to(provider, 'Details')
-    net_prov_name = view.entities.summary("Relationships").get_text_of("Network Manager")
-
-    collection = appliance.collections.network_providers
-    network_provider = collection.instantiate(prov_class=NetworkProvider, name=net_prov_name)
+    collection = appliance.collections.network_providers.filter({'provider': provider})
+    network_provider = random.choice(collection.all())
```

Comment:
**REQUIRED**: Why using random here at all, wouldn't ```[0]``` do? Do providers generally have a lot of network providers?

Reply:
@psav Yes, I can go with`[0]`, but I thought sometimes we have two clouds providers(You can say to our bad or BZ) so I thought better to go with random.

Reply:
aaaaahhh understand :) It was my bad; actually I already filtering with the specific provider :+1: